ops/tag-service/tag-tool.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import subprocess
+import re
+import semver
+
+SERVICES  = [
+    'ci-builder',
+    'ci-builder-rust',
+    'chain-mon',
+    'indexer',
+    'op-node',
+    'op-batcher',
+    'op-challenger',
+    'op-dispute-mon',
+    'op-proposer',
+    'op-ufm',
+    'proxyd',
+    'op-heartbeat',
+    'ufm-metamask',
+    'op-contracts',
+    'full', # special case for tagging op-node, op-batcher, and op-proposer together
```

Comment:
Maybe `op-stack`? I like this option, but IDK if `full` is quite the right name.

Reply:
I also like `op-stack`

Reply:
op-stack it shall be